packages/contentful/migrations/crn/researchOutputs/20230724000000-version.js

```diff
@@ -0,0 +1,63 @@
+module.exports.description = 'Change usageNotes type';
+
+module.exports.up = function (migration) {
+  const outputVersions = migration
+    .createContentType('outputVersions')
+    .name('Output Versions')
```

Comment:
I think I would prefer to use the `Research Output` prefix so that we know it belongs to that, unless there is a limit on the name length? I was looking at the content models and i couldn't find it until i opened the definition of the field inside Research Output content model